alex/corpustools/voxforge-download.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+url=http://www.repository.voxforge1.org/downloads/SpeechCorpus/Trunk/Audio/Main/16kHz_16bit
```

Comment:
Does it download all audio for all languages? Or just English? Is there a way I can choose what language I want? What about the other frequencies?